vonk/releasenotes/releasenotes.rst

```diff
@@ -17,6 +17,22 @@ Upgrading Vonk
 
 See :ref:`upgrade` for information on how to upgrade to a new version of Vonk.
 
+.. _vonk_releasenotes_340:
+
+Release 3.4.0
+-------------
+
+Feature
+^^^^^^^
+
+#. Upgraded to FHIR .NET API 1.6.0, that features a couple of changes for working with CDA logical models. See the `release notes of the API <https://github.com/FirelyTeam/fhir-net-api/releases>`_.
+#. Included the FHIR Mapper in the distribution. It is only enabled however when you include the mapping plugin in your license.
```

Comment:
Could you include a link to the FHIR Mapper docs? And maybe add a sentence that the plugin is included in the trial license?